src/Network/Network/ChangeLog.md

```diff
@@ -48,6 +48,9 @@
     - `Update-AzVirtualHub`
 * Updated cmdlets to expose two read-only properties of client certificate.
     - `Get-AzApplicationGatewayTrustedClientCertificate`
+* Updated cmdlets to add properties for new BYOIP features.
+    - `New-AzCustomIpPrefix`
```

Comment:
Please sync your code with latest network-march branch so that your change log msg can include in upcoming release section.
![image](https://user-images.githubusercontent.com/19869716/123602224-1ad08a80-d82b-11eb-96b1-8b478a1ae47b.png)